### ISSUE TITLE
Sass files Not compatible with Webpack

### ISSUE THREAD

**Issue by IAMtheIAM** [Daemonite/material#97]
1) When I do `require('daemonite-material');` - it does not bring in the scss. So the `main` script needs to be updated in the package.json with the path to scss.

2) When I do `require('daemonite-material/assets/sass/base.scss')`, webpack has compilation errors which relate to missing Fonts. When I manually add the fonts, it gives more compilation errors.

How are we supposed to use this with front end build system like Webpack and require()?

**Comment by sesemaya** [comment]
Hi @IAMtheIAM 

Issue 1 has been fixed since https://github.com/Daemonite/material/releases/tag/v4.0.0-alpha.0

Issue 2 is similar to https://github.com/Daemonite/material/issues/114. It is because Webpack does not like `url()` or `url(./)`. Since fonts are all loaded separately in v4, the only thing that can cause a problem is:
```
.waves-circle > .waves-wrap {
  -webkit-mask: url();
  mask: url();
}
```
This will be fixed once `backfilling-bootstrap-v4.0.0-alpha.6` is merged into `master`.

**Comment by IAMtheIAM** [comment]
Very nice, thanks for the update.
